example/src/Controls.tsx

```diff
@@ -28,43 +26,60 @@ export const Controls = ({ disconnect }: { disconnect: () => void }) => {
     });
   };
 
+  const toggleScreencastAndUpdateMetadata = () => {
+    toggleScreencast({
+      screencastMetadata: {
+        displayName: 'presenting',
+        type: 'screensharing',
+        active: 'true',
+      },
+      quality: Membrane.ScreencastQuality.FHD30,
+    });
+  };
+
   return (
     <View style={styles.iconsContainer}>
-      <Pressable onPress={toggleMicrophoneAndUpdateMetadata}>
-        {!isMicrophoneOn ? (
-          <Icon name="Microphone-off" size={iconSize} />
-        ) : (
-          <Icon name="Microphone" size={iconSize} />
-        )}
-      </Pressable>
-      <Pressable onPress={toggleCameraAndUpdateMetadata}>
-        {!isCameraOn ? (
-          <Icon name="Cam-disabled" size={iconSize} />
-        ) : (
-          <Icon name="Cam" size={iconSize} />
-        )}
-      </Pressable>
-      <Pressable onPress={disconnect}>
-        <Icon name="Hangup" size={iconSize} />
-      </Pressable>
-      <Pressable
-        onPress={() =>
-          toggleScreencast({
-            screencastMetadata: {
-              displayName: 'presenting',
-              type: 'screensharing',
-              active: 'true',
-            },
-            quality: Membrane.ScreencastQuality.FHD30,
-          })
-        }
-      >
-        {isScreencastOn ? (
-          <Icon name="Screenshare" size={iconSize} />
-        ) : (
-          <Icon name="Screen-off" size={iconSize} />
-        )}
-      </Pressable>
+      {!isMicrophoneOn ? (
+        <InCallButton
+          type="primary"
+          iconName="Microphone-off"
+          onPress={toggleMicrophoneAndUpdateMetadata}
+        />
+      ) : (
+        <InCallButton
+          type="primary"
+          iconName="Microphone"
+          onPress={toggleMicrophoneAndUpdateMetadata}
+        />
+      )}
+      {!isCameraOn ? (
+        <InCallButton
+          type="primary"
```

Comment:
we can make the primary one default = less typing

Reply:
Done.